examples/objectServerExample/src/main/java/io/realm/examples/objectserver/CounterActivity.java

```diff
@@ -86,12 +121,21 @@ public void onChange(CRDTCounter counter) {
                 }
             });
             counterView.setText("0");
+
+            // Setup progress listeners for indeterminate progress bars
+            session = SyncManager.getSession(config);
+            session.addDownloadProgressListener(ProgressMode.INDEFINITELY, downloadListener);
+            session.addUploadProgressListener(ProgressMode.INDEFINITELY, uploadListener);
         }
     }
 
     @Override
     protected void onStop() {
         super.onStop();
+        if (session != null) {
+            session.removeProgressListener(downloadListener);
```

Comment:
The are being added in `onStart`? https://github.com/realm/realm-java/pull/4415/files#diff-f293d30335c94eee737ce7a8c1fd2790R127